roadmap.md

```diff
@@ -20,6 +20,9 @@ info@openrobotics.org.
     * [All open tickets](https://github.com/search?q=org%3Aignitionrobotics+label%3A%22scripting%22&state=open&type=Issues)
     * [Status](https://github.com/orgs/ignitionrobotics/projects/3?card_filter_query=label%3A%22scripting%22)
 
+## 2021 Q3 (Jul - Sep) - ongoing
+* **New GUI widgets in Ignition Gazebo**: New widgets that allow the users to debug their simulations and models. For example: visualize joints, wireframe, CoM, inertia, etc.
```

Comment:
Let's move these items to the feature roadmap below. This section is reserved for the quarterly roadmaps, which involve a set of features, like the ones above.